build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
   id "idea"
   id "jacoco" // Java Code Coverage plugin
   id "com.github.ben-manes.versions" version "0.42.0"
-  id "com.github.spotbugs" version "5.0.9" apply false
+  id "com.github.spotbugs" version "5.1.5" apply false
```

Comment:
seems to be that the latest is 6.0.6. Would you please use that?
https://plugins.gradle.org/plugin/com.github.spotbugs